mobile/lib/firebase_options.dart

```diff
@@ -66,7 +66,7 @@ class DefaultFirebaseOptions {
     databaseURL: 'https://airqo-250220.firebaseio.com',
```

Comment:
We have a couple of exposed values in the `FirebaseOptions` function that I feel would be best fit if added to the .env file. Could we review this approach of hard coding them before moving to production?
cc @Mnoble-19 @BenjaminSsempala @NoahNsimbe